web/template/layout/error-summary.gohtml

```diff
@@ -6,7 +6,7 @@
       </h2>
       <div class="govuk-error-summary__body">
         <ul class="govuk-list govuk-error-summary__list">
-          {{ if .Detail }}
+          {{ if (and .Detail (ne .Detail "Payload failed validation") )}}
```

Comment:
I opted to leave Detail property as I think it's required for an error message for the create relationship form 
https://github.com/ministryofjustice/opg-sirius-lpa-frontend/pull/45